code/game/turfs/simulated/walls.dm

```diff
@@ -139,7 +139,7 @@
 	P.setAngle(new_angle_s)
 	return TRUE
 
-/turf/simulated/wall/proc/dismantle_wall(devastated = 0, explode = 0)
+/turf/simulated/wall/dismantle_wall(devastated = FALSE, explode = FALSE)
```

Comment:
Can you make this return TRUE if successful please? I'm working a fix that's very closely related to this (involves crowbars and reinforced walls) and it would be nice to display a message only if it's successful

Reply:
will do!